tests/test_noisemodel.py

```diff
@@ -132,20 +138,20 @@ def test_general_noisemodel(backend):
     value1 = sample_expectation_ps_noisfy(c, x=[0, 1], noise_conf=noise_conf, nmc=10000)
     value2 = c.sample_expectation_ps(x=[0, 1], noise_conf=noise_conf, nmc=10000)
     value3 = dmc.sample_expectation_ps(x=[0, 1], noise_conf=noise_conf)
-    np.testing.assert_allclose(value1, value2, atol=1e-1)
-    np.testing.assert_allclose(value3, value2, atol=1e-1)
+    np.testing.assert_allclose(value1, value2, atol=1e-2)
+    np.testing.assert_allclose(value3, value2, atol=1e-2)
```

Comment:
a stricter checker for sample expectation may fail the test in rare case, unless the strict criteria is still rather flexible

Reply:
Then I think it's better to increase the number of runs in MC simulations. `1e-1` is too lose IMO

Reply:
you can check the value now, I think the criteria can be one order larger than the average difference in trials

Reply:
`nmc=100000` reduces the std deviation to around 1e-3

Reply:
then I think it is ok